packages/react-components/react-combobox/library/src/components/Combobox/useCombobox.tsx

```diff
@@ -112,6 +115,15 @@ export const useCombobox_unstable = (props: ComboboxProps, ref: React.Ref<HTMLIn
     ...baseState,
   };
 
+  const { targetDocument } = useFluent();
+
+  useOnClickOutside({
```

Comment:
@smhigley  now that outside click handling is a requirement, is there still a need to handle open/close with focus/blur?